rgw/v2/tests/s3_swift/test_swift_basic_ops.py

```diff
@@ -51,6 +51,7 @@
 import traceback
 
 import names
+import requests
 import v2.lib.manage_data as manage_data
 import v2.lib.resource_op as swiftlib
 import v2.utils.utils as utils
```

Comment:
Please update the config file : configs/test_swift_at_root.yaml

Reply:
Done , pls review